src/Illuminate/Console/Scheduling/Schedule.php

```diff
@@ -97,6 +97,8 @@ public function job($job, $queue = null, $connection = null)
             $job = is_string($job) ? resolve($job) : $job;
 
             if ($job instanceof ShouldQueue) {
+                $queue = $queue !== null ? $queue : $job->queue;
```

Comment:
`$queue = $queue ?? $job->queue`

Reply:
Is an empty string a valid queue name?

Reply:
Never mind, I had a brain-fart. I was thinking of the `?:` operator. `??` seem to work as expected.